Source/WebCore/page/TextIndicator.cpp

```diff
@@ -69,30 +70,43 @@ Ref<TextIndicator> TextIndicator::create(const TextIndicatorData& data)
 
 RefPtr<TextIndicator> TextIndicator::createWithRange(const SimpleRange& range, OptionSet<TextIndicatorOption> options, TextIndicatorPresentationTransition presentationTransition, FloatSize margin)
 {
-    RefPtr frame = range.startContainer().document().frame();
+    auto rangeToUse = range;
```

Comment:
Nit: `auto&`.

Reply:
I tried this, but I don't think it's possible since `range` is `const`, and then `rangeToUse` is reassigned later on, so it won't compile